### ISSUE TITLE
Documentation: SCC set up instructions are hard to follow

### ISSUE THREAD

**Issue by mjainQA** [Komodo/KomodoEdit#2691]
### Short Summary
The current SCC documentation is difficult to follow. It needs improvement.

### Steps to Reproduce

Go to http://docs.komodoide.com/Manual/scc#scc_commands

### Expected results

Simple and easy to follow instructions are needed with screenshots.

### Platform Information

*Komodo Edit or IDE?* IDE
*Komodo Version?* 10.2
*Operating System (and version)?* macOS

**Comment by cgchoffman** [comment]
Those aren't setup instructions.  Those are commands that you can run once you're setup.  What exactly are you looking for? How to have a repo Komodo?  Perhaps this? http://docs.komodoide.com/Manual/scc#source-code-control-komodo-ide-only_scc-checkout-wizard

**Comment by mjainQA** [comment]
For e.g.
 Doc mentions: 

1. "Access common source code control commands from the SCC Toolbar at the top of the Komodo workspace." - I am unable to find so.

2. "Invoke source code control context menus by right-clicking files or folders in the following areas:

Projects sidebar
Toolbox
Editor Pane"

Is it the project in the places or projects pane? Right clicking on project folder presents SCC different options vs file. Right clicking on the file in the editor has different SCC menu options. What is a Toolbar?

3. " Click Checkout to run the command. Current status and command output will be displayed as the command runs." - It does not specify how to set up user name for git.

Having screenshots would be helpful.

**Comment by cgchoffman** [comment]
Oh I thought you were talking specifically about setting up SCC.  I changed the title slightly to make it clear that the whole thing needs to be reworked.  Screenshots would definitely be helpful.
